python/py-pytest-remotedata/Portfile

```diff
@@ -17,15 +17,16 @@ long_description    ${description}
 
 homepage            https://astropy.org
 
-checksums           rmd160  01a774726a884c1dba440757e185a75156a74707 \
-                    sha256  e20c58d4b7c359c4975dc3c3d3d67be0905180d2368be0be3ae09b15a136cfc0 \
-                    size    11583
+checksums           rmd160  59af296ed687cf750b7cc067367509f7756a6049 \
+                    sha256  66920bf1c62928b079d0e611379111a0d49f10a9509ced54c8269514ccce6ee3 \
+                    size    12718
 
 python.versions     38 39 310
 
 if {${name} ne ${subport}} {
     depends_build-append \
-                    port:py${python.version}-setuptools
+                    port:py${python.version}-setuptools \
+                    port:py${python.version}-setuptools_scm
 
     depends_lib-append \
```

Comment:
these dependencies don't look right - [upstream](https://github.com/astropy/pytest-remotedata/blob/3e82525eaeed217b0782a7a7e22c49253acb9d71/setup.cfg#L33) doesn't list `six` but has `packaging`

Reply:
@reneeotten very good catch! `six` was removed before 0.3.3 https://github.com/astropy/pytest-remotedata/commit/4b2fa740f4d3ab9007ce2893fcc79e5de3c6329b and `packaging` was added at the same release: https://github.com/astropy/pytest-remotedata/commit/8acba7e912999b6bb73314a22a7589ff80a31bc9